apps/cowswap-frontend/src/modules/limitOrders/hooks/useHandleOrderPlacement.ts

```diff
@@ -18,12 +18,29 @@ import { useCloseReceiptModal } from 'modules/ordersTable/containers/OrdersRecei
 import { TradeConfirmActions } from 'modules/trade/hooks/useTradeConfirmActions'
 import { useAlternativeOrder, useHideAlternativeOrderModal } from 'modules/trade/state/alternativeOrder'
 import { getSwapErrorMessage } from 'modules/trade/utils/swapErrorHelper'
+import { useTradeFlowAnalytics } from 'modules/trade/utils/tradeFlowAnalytics'
```

Comment:
Should be `import { useTradeFlowAnalytics } from 'modules/trade'`

Reply:
Addressed